### ISSUE TITLE
kbox update 0.10.4 PDO issue

### ISSUE THREAD

**Issue by labboy0276** [kalabox/kalabox#601]
When I run ```kbox update``` on the new binary && then run an app, I get this error:

```
Warning: PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or service not known in /code/includes/database/database.inc on line 307 Call Stack: 0.3113 4657008 1. _drupal_exception_handler() /code/includes/bootstrap.inc:0 0.3114 4658336 2. _drupal_log_error() /code/includes/bootstrap.inc:2358 0.3115 4658368 3. drupal_maintenance_theme() /code/includes/errors.inc:179 0.3128 4726560 4. _drupal_maintenance_theme() /code/includes/bootstrap.inc:2634 0.3378 8204952 5. list_themes() /code/includes/theme.maintenance.inc:57 0.3379 8205856 6. _system_rebuild_theme_data() /code/includes/theme.inc:798 0.3538 8213208 7. module_invoke_all() /code/modules/system/system.module:2507 0.3538 8213640 8. module_implements() /code/includes/module.inc:892 0.3538 8214496 9. cache_get() /code/includes/module.inc:723 0.3538 8214496 10. DrupalDatabaseCache->get() /code/includes/cache.inc:56 0.3538 8214800 11. DrupalDatabaseCache->getMultiple() /code/includes/cache.inc:329 0.3538 8214800 12. db_escape_table() /code/includes/cache.inc:348 0.3538 8214800 13. Database::getConnection() /code/includes/database/database.inc:2570 0.3538 8214880 14. Database::openConnection() /code/includes/database/database.inc:1479 0.3538 8218848 15. DatabaseConnection_mysql->__construct() /code/includes/database/database.inc:1689 0.3539 8220232 16. DatabaseConnection->__construct() /code/includes/database/mysql/database.inc:55 0.3539 8221496 17. PDO->__construct() /code/includes/database/database.inc:307 0.4094 8230736 18. _drupal_error_handler() /code/includes/database/database.inc:307 0.4094 8230736 19. _drupal_error_handler_real() /code/includes/bootstrap.inc:2340 0.4097 8249216 20. _drupal_log_error() /code/includes/errors.inc:75 0.4097 8249416 21. watchdog() /code/includes/errors.inc:202 0.4098 8251632 22. module_implements() /code/includes/bootstrap.inc:1760 0.4098 8251896 23. cache_get() /code/includes/module.inc:723 0.4098 8251896 24. DrupalDatabaseCache->get() /code/includes/cache.inc:56 0.4098 8252200 25. DrupalDatabaseCache->getMultiple() /code/includes/cache.inc:329 0.4098 8252200 26. db_escape_table() /code/includes/cache.inc:348 0.4098 8252200 27. Database::getConnection() /code/includes/database/database.inc:2570 0.4098 8252280 28. Database::openConnection() /code/includes/database/database.inc:1479 0.4099 8256248 29. DatabaseConnection_mysql->__construct() /code/includes/database/database.inc:1689 0.4099 8257632 30. DatabaseConnection->__construct() /code/includes/database/mysql/database.inc:55 0.4099 8258896 31. PDO->__construct() /code/includes/database/database.inc:307 Warning: PDO::__construct(): [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known (trying to connect via tcp://cor777.kbox:3306) in /code/includes/database/database.inc on line 307 Call Stack: 0.3113 4657008 1. _drupal_exception_handler() /code/includes/bootstrap.inc:0 0.3114 4658336 2. _drupal_log_error() /code/includes/bootstrap.inc:2358 0.3115 4658368 3. drupal_maintenance_theme() /code/includes/errors.inc:179 0.3128 4726560 4. _drupal_maintenance_theme() /code/includes/bootstrap.inc:2634 0.3378 8204952 5. list_themes() /code/includes/theme.maintenance.inc:57 0.3379 8205856 6. _system_rebuild_theme_data() /code/includes/theme.inc:798 0.3538 8213208 7. module_invoke_all() /code/modules/system/system.module:2507 0.3538 8213640 8. module_implements() /code/includes/module.inc:892 0.3538 8214496 9. cache_get() /code/includes/module.inc:723 0.3538 8214496 10. DrupalDatabaseCache->get() /code/includes/cache.inc:56 0.3538 8214800 11. DrupalDatabaseCache->getMultiple() /code/includes/cache.inc:329 0.3538 8214800 12. db_escape_table() /code/includes/cache.inc:348 0.3538 8214800 13. Database::getConnection() /code/includes/database/database.inc:2570 0.3538 8214880 14. Database::openConnection() /code/includes/database/database.inc:1479 0.3538 8218848 15. DatabaseConnection_mysql->__construct() /code/includes/database/database.inc:1689 0.3539 8220232 16. DatabaseConnection->__construct() /code/includes/database/mysql/database.inc:55 0.3539 8221496 17. PDO->__construct() /code/includes/database/database.inc:307 0.4094 8230736 18. _drupal_error_handler() /code/includes/database/database.inc:307 0.4094 8230736 19. _drupal_error_handler_real() /code/includes/bootstrap.inc:2340 0.4097 8249216 20. _drupal_log_error() /code/includes/errors.inc:75 0.4097 8249416 21. watchdog() /code/includes/errors.inc:202 0.4098 8251632 22. module_implements() /code/includes/bootstrap.inc:1760 0.4098 8251896 23. cache_get() /code/includes/module.inc:723 0.4098 8251896 24. DrupalDatabaseCache->get() /code/includes/cache.inc:56 0.4098 8252200 25. DrupalDatabaseCache->getMultiple() /code/includes/cache.inc:329 0.4098 8252200 26. db_escape_table() /code/includes/cache.inc:348 0.4098 8252200 27. Database::getConnection() /code/includes/database/database.inc:2570 0.4098 8252280 28. Database::openConnection() /code/includes/database/database.inc:1479 0.4099 8256248 29. DatabaseConnection_mysql->__construct() /code/includes/database/database.inc:1689 0.4099 8257632 30. DatabaseConnection->__construct() /code/includes/database/mysql/database.inc:55 0.4099 8258896 31. PDO->__construct() /code/includes/database/database.inc:307```

**Comment by labboy0276** [comment]
Doing a ```kbox rebuild``` fixes the issue.

**Comment by pirog** [comment]
there must be an event or somesuch that gets called during a rebuild command vs hitting the rebuild func directly that accounts for this.

**Comment by pirog** [comment]
Hmmm i just tried doing a rebuild and i was unable to replicate this.

**Comment by pirog** [comment]
This should be resolved in the 0.11.0 release on Friday
